backend/apis/leads/list-leads.api.js

```diff
@@ -29,6 +29,14 @@ app.get("/api/leads", (req, res, next) => {
     );
   }
 
+  let whereClause = `WHERE leads.isDeleted = false`;
+  let whereClauseValues = [];
+
+  if (req.query.name) {
```

Comment:
Let's update [the api documentation](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/leads.md) to have the `name` query parameter there.